### PR TITLE
Update user.service.ts

### PR DESCRIPTION
**Pull request by mir-ishfaq-hussain**
###Description
i have made changes to the convertDbUserToUser function removed the unnecessary ternary operator and introduced simple null coalescing operator  while returning instead of ternary operator

Closes # 

### Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

**Review by balub — APPROVED** [review]
Please make sure the lastest commits are present in the PR.

**Comment by mirishfaq-logicparadise** [comment]
okay will do the same

**Review by mirarifhasan — APPROVED** [review]
LGTM

File: packages/hoppscotch-backend/src/user/user.service.ts [hunk 1]
```diff
@@ -41,12 +41,8 @@ export class UserService {
...
     return {
       ...dbUser,
-      currentRESTSession: dbCurrentRESTSession
-        ? JSON.stringify(dbCurrentRESTSession)
-        : null,
-      currentGQLSession: dbCurrentGQLSession
-        ? JSON.stringify(dbCurrentGQLSession)
-        : null,
+      currentRESTSession: JSON.stringify(dbCurrentRESTSession) ?? null,
+      currentGQLSession:  JSON.stringify(dbCurrentGQLSession) ?? null,
```
Comment (mirarifhasan) [inline packages/hoppscotch-backend/src/user/user.service.ts:45]: `dbCurrentRESTSession` can be null.
`JSON.stringify(null)` will return `"null"` as string data-type, which breaks the contract.

Same goes for `dbCurrentGQLSession`.

**Review by mirarifhasan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**